src/options.h

```diff
@@ -57,12 +61,24 @@ void set_default_values_from_env()
             rotated_files = 5;
         }
     }
+
+    if ( log_directory ==  NULL ) {
+        env_val = g_getenv("LOGPROXY_LOG_DIRECTORY");
+        if ( env_val != NULL ) {
+            log_directory = (gchar *)env_val;
+        }
+        else {
+            getcwd(cwd, sizeof(cwd));
```

Comment:
use g_get_current_dir()
https://developer.gnome.org/glib/stable/glib-Miscellaneous-Utility-Functions.html#g-get-current-dir

instead of old getcwd?